extensions/positron-python/package.json

```diff
@@ -557,37 +557,49 @@
                     "default": "python",
                     "markdownDescription": "%python.defaultInterpreterPath.description%",
                     "scope": "machine-overridable",
```

Comment:
Should we change this to `machine`? I hesitated to change this from the upstream default, even though changing it to `machine` would be more consistent with the rest of the interpreter settings.

`machine-overridable` means that in Web, the user can set this at the Workspace level. `machine` means that in Web, the user can only set this at the Remote level.

Since this is a machine-dependent path, I think it makes more sense to use `machine`, but not sure if we want to deviate from upstream on this setting.